src/senaite/core/setuphandlers.py

```diff
@@ -504,14 +504,7 @@ def pre_install(portal_setup):
 
     :param portal_setup: SetupTool
     """
-    logger.info("SENAITE CORE pre-install handler [BEGIN]")
-
-    # https://docs.plone.org/develop/addons/components/genericsetup.html#custom-installer-code-setuphandlers-py
-    profile_id = PROFILE_ID
-    context = portal_setup._getImportContext(profile_id)
-    portal = context.getSite()  # noqa
```

Comment:
the code above initialized `portal` which wasn't used, so this was dead code

we could also remove `pre_install()` but I thought it may be overridden in some customer code